pandas/io/pytables.py

```diff
@@ -4875,8 +4876,16 @@ def _unconvert_index(
 
 
 def _maybe_convert_for_string_atom(
-    name: str, block, existing_col, min_itemsize, nan_rep, encoding, errors
+    name: str,
+    block,
+    existing_col,
+    min_itemsize,
+    nan_rep,
+    encoding,
+    errors,
+    block_columns: List[str] = [],
 ):
+    # block_columns(list[str]): the label of columns for debug info use.
```

Comment:
can you add a doc-string template e.g. Parameter / Returns (just list the arg names, and document the new one)

Reply:
got it.
but giving a private function-which is only internally called once-a docstring looks a bit deluxe...
will add docstring.